### ISSUE TITLE
[Adaptive Cards-FoodOrder.Json]: Screen reader is not announcing the 'Expand/Collapse' state of "Steak", "Chicken" and "Tofu" buttons under 'FoodOrder.json.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9241]
### Target Platforms

UWP

### SDK Version

1.0.3.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Visualizer
OS Version: 25H2 (OS Build 26200.5670) 
App Version: 1.0.3.0
Screen Reader: Narrator/NVDA

**Repro steps:**
1. Turn on Narrator through Ctrl + Windows + Enter key.
2. Open the Adaptive Card Visualizer application.
3. Navigate to FoodOrder.Json button by using Tab key and activate it.
4. FoodOrder adaptive card will be appeared on right sight of the pane.
5. Navigate to FoodOrder card by using Tab key.
6. Navigate to "Steak", "Chicken" and "Tofu" buttons with Tab key and observe the issue.

**Actual Result:**
Screen reader is not announcing the 'Expand/Collapse' state of "Steak", "Chicken" and "Tofu" buttons under 'FoodOrder.json. Screen Reader is only announcing 'Steak button'.

**Observation:** 
Same issue replicates with NVDA.

**Expected Result:**
Screen reader should announce the 'Expand/Collapse' state of "Steak", "Chicken" and "Tofu" buttons under 'FoodOrder.json. Screen reader should announce as, "Steak button Expanded/Collapsed".

**User Impact:**
User depending on screen reader might get affected if screen reader does not announce state information about the control. Screen reader user will not be able to know the property of that control and will face problem while interacting with it.

**WCAG Reference link:**
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/b9b83223-7bb9-4f0c-9e65-aefcf3bc3592

![Image](https://github.com/user-attachments/assets/458f83ec-3c00-43bf-85f5-7b2647a22d8a)

### Card JSON

```JSON
N/A
```

**Comment by Rishikesh6521** [comment]
Original Bug Id: #7626
